marrow/mailer/message.py

```diff
@@ -258,7 +258,8 @@ def mime(self):
 		return message
 
 	def attach(self, name, data=None, maintype=None, subtype=None,
-		inline=False, filename=None, encoding=None):
+		inline=False, filename=None, filename_charset='', filename_language='',
+		content_encoding=None):
```

Comment:
Unsure of why this needed to be a breaking change?  (`encoding` → `content_encoding`)  Why write more and needlessly break existing use?